IdlImpTool/IDLImp.csproj

```diff
@@ -1,7 +1,7 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net461</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;net5.0</TargetFrameworks>
```

Comment:
Why `net5` and not `net6`? At this point only `netcoreapp3.1` is still in support...

Reply:
Because I didn't have VS2022 on my machine yet. :( And neither does Appveyor..

Reply:
https://github.com/sillsdev/IdlImporter/issues/22